src/SessionSetup/include/SessionSetup/RetrieveInstances.h

```diff
@@ -20,6 +20,11 @@ namespace orbit_session_setup {
 
 class RetrieveInstances {
```

Comment:
Not necessarily part of this PR, but I've just realized that a comment is missing here. According to our style guide, each class should have a brief description.

Reply:
I added a comment